module-apps/application-messages/windows/NewMessage.cpp

```diff
@@ -312,7 +344,8 @@ namespace gui
         };
         body->addWidget(message);
         body->addWidget(new Span(Axis::Y, body->getHeight()));
-        message->setText(""); // to set actual size of Text
+        recipient->setText(""); // to set actual size of Text
```

Comment:
Is this comment sth like TO DO marker or the text size is 0?

Reply:
This is a comment of the code I found here. I will change it to a more appropriate one